web/styles/rendered_markdown.css

```diff
@@ -397,6 +397,64 @@
         margin-right: 5px;
     }
 
+    /* In browsers that support `:has()`, we pull
+       `.rendered_markdown` out of the baseline group
+       formed with EDITED/MOVED markers and the
+       timestamp when the first child of the rendered
+       markdown is a media element. */
+    &:has(> .message_inline_image:first-child) {
+        align-self: center;
+    }
```

Comment:
Hmm, why is it just media elements that logic should apply to? (E.g. would a spoiler have the same problem)?

Reply:
Spoilers and other widgets won't have this problem, so long as they have a text element that's part of the document flow.

That said, the EDITED/MOVED markers and timestamps do appear slightly lower in the message box on elements like spoilers and even code blocks, just because those elements themselves have first lines of text that are also lower in the message box.

I would not be in favor of applying this solution to those elements at present, though, because the vertical alignment of the markers and timestamps will look off and arbitrary.

My longer-term plan, as part of handling information-density in the message area, is to specify more exacting line-heights throughout that area, and then use those to be able to more uniformly present timestamps in relation to widgets and more typical messages that just open with paragraph text.